src/Commands/LightsOn.h

```diff
@@ -0,0 +1,21 @@
+#ifndef LightsOn_H
+#define LightsOn_H
+
+#include "Commands/Command.h"
+#include "../Robot.h"
+
+class LightsOn: public Command
+{
+public:
+	LightsOn(Lights::LightValue value);
+	void Initialize();
+	void Execute();
+	bool IsFinished();
+	void End();
+	void Interrupted();
+
+private:
+	Lights::LightValue mValue;
```

Comment:
I believe you want to include Lights.h, to make it clear what your class depends on by quick glance at the includes